### DISCUSSION TITLE
Person Notifications

### DISCUSSION THREAD

**Discussion by alardierejr** [dgreif/ring#848] [Q&A]
Hey!

Is there a way to setup the plugin so that I will only get person notifications on HomeKit vs are motion detections?   Motion detection are overkill and the new ring floodlights notify you if people are detected as well.  In the ring app you can turn off motion detection notifications and only get the person notifications.  

This would be a great feature to have so that it works better with HomeKit.  Let me know!

Thank you!

**Comment by dgreif** [comment]
See https://github.com/dgreif/ring/issues/263#issuecomment-667135126

**Reply by alardierejr** [reply]
Hello,

Unfortunately this is not the case. I'm new to Homebridge as well as Ring but I am getting notifications to HomeKit for any motion, not just people. My settings in ring are setup to only receive person notifications for all 4 spotlights. 

If you had done extensive testing last year, I'd say something has changed. 

Thanks

**Reply by alardierejr** [reply]
@dgreif did you see my reply?  I see you thumbs uped it but do you realize it is not functioning properly?  Should I reach out to HomeBridge instead?  Are you going to create a defect or enhancement request for this?

Thanks

**Reply by dgreif** [reply]
I see your reply, but have not had time to look into it. The setting you referred to in the Ring app is device specific, so only that one phone is limited to person notifications. Allowing that to be configured per camera in the homebridge plugin is not trivial

**Reply by alardierejr** [reply]
> See [#263 (comment)](https://github.com/dgreif/ring/issues/263#issuecomment-667135126)

Thanks.  I just wanted to be sure you were aware that the comment you referenced #263 is unfortunately not valid or something changed.  As of right now I won't use notifications because you can imagine the amount you'd get every time there's some type of motion.  Once you do take a look, is there a way for me to follow the issue/fix?  Just wondering - new to this website.

**Reply by dgreif** [reply]
You can just click the "Subscribe" button on #263 and then you will get notifications if any details are added there
